### DISCUSSION TITLE
Consider adding teams?

### DISCUSSION THREAD

**Discussion by Explosion-Scratch** [ScratchAddons/ScratchAddons#7] [Discussion]
So I didn't want to just go ahead and do this, but anyways what do you guys think of making teams? Here are my ideas:
• Coders (Implement userstyles, work on the addon, fix glitches)
    • Bug fixers
    • Contributors
• Management (World_Languages and any others who mainly manage this and coordinate stuff)
• Helpers (People who archive and collect userscripts but are not intrested in the coding aspect)
    • Userscript collectors.
    • Archivers. (I think this is nessecary because archive.org takes SO long to archive stuff and there are so many pages per userscript to archive.)

 I think that even though teams are not totally nessecary right now they would definitely help keep things organized and allow other people to join easily.

**Comment by Explosion-Scratch** [comment]
What? I assigned myself?! Also my list didn't work right. I put indents in places :| I am a n00b

**Comment by tb148** [comment]
My idea (I merged some):

- Programmers
- Testers (= Bug fixers)
- Managers
- Collectors (For userscripts)
- Archivers (Is it necessary?)

Being in multiple teams is allowed, so helpers=collectors+archivers is removed. And I don't get why there is "contributors".

**Comment by BoomerScratch** [comment]
Instead of making archives, use branches or [releases](https://docs.github.com/en/github/administering-a-repository/managing-releases-in-a-repository).

**Comment by Explosion-Scratch** [comment]
@tb148 Thanks! That looks a lot better! Coders would've been split into two subcategories, bug fixers and userscript collectors and archivers woul've fallen under helpers. My indents didn't work. 

@BoomerScratch The problem with this is that a lot of userscripts are not on github. I'll see if I can somehow archive multiple pages at once on the internet archive.

**Comment by tb148** [comment]
Note: Since @Explosion-Scratch Seems to unintentionally assign himself, I cleared his assignment.

**Comment by tb148** [comment]
> @tb148 Thanks! That looks a lot better! Coders would've been split into two subcategories, bug fixers and userscript collectors and archivers would've fallen under helpers. My indents didn't work.
What are the two categories for coders?

**Comment by Explosion-Scratch** [comment]
Here's the original list as I had wanted it.
1) Coders:
    a) Contributors- contribute to the code
    b) Testers and bug fixers.
2) Management
3) Helpers
    a) Archivers
    b) Userscript collectors

**Comment by tb148** [comment]
@Explosion-Scratch I think that bug fixers should be merged into contibutors.

**Comment by Explosion-Scratch** [comment]
@tb148 Sure!

**Comment by azureitte** [comment]
Do we need teams though? This is a fairly small project and I think there's no need in assigning roles to people depending on what they want to do.

**Comment by WorldLanguages** [comment]
Yeah agreed. Writing readmes or having teams at this stage might be fun but not necessary
